README.md

```diff
@@ -1,87 +1,107 @@
-# heat-docker-kubernetes
+# Getting started with k8s-docker-provisioner
 
-Steps:
+This guide will take you through the steps of deploying Kubernetes to Openstack using docker. The primary concept is described here: [docker-multinode](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/kubernetes/blob/master/docs/getting-started-guides/docker-multinode/master.md)
 
-1. Deploy a Kubernetes cluster on DevStack using Heat
+The default OS is CentOS 7.
```

Comment:
Please remove CentOS from this doc. Instead we should explain the Docker bootstrap concept. But I would suggest to do this task later